From 310b6a26f58814568245c781c2136504d1a6f338 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date: Sat, 17 Mar 2018 18:41:56 +0100
Subject: [PATCH] Fix build with Qt 5.11 (missing QAbstractItemView)

Reviewers: mlaurent

Reviewed By: mlaurent

Subscribers: #kde_pim

Tags: #kde_pim

Differential Revision: https://phabricator.kde.org/D11416
---
 src/widgets/manageaccountwidget.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/widgets/manageaccountwidget.cpp b/src/widgets/manageaccountwidget.cpp
index 7875081c8..9a6f0873a 100644
--- a/src/widgets/manageaccountwidget.cpp
+++ b/src/widgets/manageaccountwidget.cpp
@@ -32,6 +32,7 @@
 #include <KWindowSystem>
 #include <QKeyEvent>
 #include <QAbstractItemDelegate>
+#include <QAbstractItemView>
 
 using namespace Akonadi;
 
